apps/testingaccessibility/src/components/app/footer.tsx

```diff
@@ -39,6 +39,9 @@ const Footer: React.FC<FooterProps> = () => {
                 <li>
                   <NavLink href="/articles">Articles</NavLink>
                 </li>
+                <li>
+                  <NavLink href="/newsletter">Newsletter</NavLink>
+                </li>
```

Comment:
suggestion: This could be conditional based on whether `ck_subscriber_id` is present.